app/utils/RegexConstants.scala

```diff
@@ -43,13 +43,16 @@ trait RegexConstants {
   final val nonApiNameRegex             = """^[a-zA-Z0-9 "'&,\-\\\/]*$"""
   final val orgNameRegex                = """^[a-zA-Z0-9 &`\-\'\\\^]*$"""
   final val utrRegex                    = "^[0-9]*$"
+  final val giinRegex                   = "^[a-zA-Z0-9]{6}\\.[a-zA-Z0-9]{5}\\.[a-zA-Z0-9]{2}\\.[a-zA-Z0-9]{3}$"
+  final val giinFormatRegex             = "^[A-NP-Z0-9]{6}\\.[A-Z0-9]{5}\\.[A-Z0-9]{2}\\.[A-Z0-9]{3}$"
```

Comment:
I don't think this is strict enough? There are a lot more rules on what makes a GIIN valid.

For example: there can't be a letter O in the second group of 5 characters, and the third group of two characters is very limited in what is allowed in it.